code/modules/xenomorph/xeno_structures.dm

```diff
@@ -133,9 +133,8 @@
 	if(!do_after(user, 20, FALSE, victim, BUSY_ICON_DANGER) || QDELETED(src))
 		return
 
-	victim.chestburst = 2 //So you can't reuse corpses if the silo is destroyed
-	victim.update_burst()
 	victim.forceMove(src)
+	victim.gib() //Gibing is much easier for monitor item tracking, since it unequip everything. Not a perfect fit, but works fine for now
```

Comment:
Bit on the fence about this
Upsides:
+Less mobs to process at highpop
Downsides:
-Theres less of the shock effect when a silo pops and a ton of people fly out

I'll leave the choice up to you here though

Reply:
The gore is fun for xenos. We can add a gore effect on silo depending on how much they were used, even spawning hands or shit if needed

Reply:
Currently the item stays out of the silo, i need to fix this i think, it's not really good